saw-remote-api/src/SAWServer/Data/SetupValue.hs

```diff
@@ -44,7 +44,7 @@ instance FromJSON cryptolExpr => FromJSON (CrucibleSetupVal cryptolExpr) where
           TagNullValue -> pure NullValue
           TagCryptol -> CryptolExpr <$> o .: "expression"
           TagArrayValue -> ArrayValue <$> o .: "elements"
-          TagStructValue -> StructValue <$> o .: "fields"
+          TagTupleValue -> TupleValue <$> o .: "elements"
```

Comment:
I know that this isn't the officially sanctioned source of documentation anymore, but I'd find it helpful if [`old-SAW.rst`](https://github.com/GaloisInc/saw-script/blob/665a08ff23b2740320d7ceb04bb8e183f625a6be/saw-remote-api/docs/old-Saw.rst) were updated to reflect this change to the SAW remote API, especially in light of #1121.